cwltool/utils.py

```diff
@@ -9,8 +10,11 @@
 import pkg_resources
 from functools import partial  # pylint: disable=unused-import
 from typing import (IO, Any, AnyStr, Callable,  # pylint: disable=unused-import
-                    Dict, Iterable, List, Optional, Text, Tuple, TypeVar,
-                    Union)
+                    Dict, Iterable, List, Optional, Text, Tuple, TypeVar, Union,
+                    TYPE_CHECKING)
+
+if TYPE_CHECKING:
+    from typing import Deque
```

Comment:
Ugh, so annoying that appveyor's Pyhon 3.5 is 3.5.3 and not 3.5.4!

Looks like the canonical way to fix this is to use https://pypi.org/project/typing-extensions/ and
`from typing_extensions import Deque`

Reply:
Done, hopefully I added the requirement everywhere I needed to.